### ISSUE TITLE
Magisk is lagging in the OxygenOS 14, without it, it is working fine

### ISSUE THREAD

**Issue by kouzelnik3** [topjohnwu/Magisk#7655]
Magisk is causing the whole OxygenOS 14 lags. Without the Magisk, it is working fine. The bahavior is the same with or without the modules on. 

The same applies for me on OnePlus 10 Pro.

Device: OnePlus Pad and OnePlus 10 Pro
Android version: 14
Magisk version name: d7750b72
Magisk version code: 26403
[magisk_log_2023-12-25T16.04.20.log](https://github.com/topjohnwu/Magisk/files/13766556/magisk_log_2023-12-25T16.04.20.log)

**Comment by NorthVex** [comment]
I am experiencing the same behavior and additionally, there are occasional screen flickers in the form of rapid screen brightness decrease and increase in seconds but it doesnt go all the way black. I also tried disabling the "virtual ram" option but this behavior continued. Only when I reflashed the phone went back to normal. 

Device: Oneplus Nord 3 5G (16/256GB)
Android Version: 14
Magisk App Version: 26.4 (latest)
